fairlearn/reductions/_exponentiated_gradient/_exponentiated_gradient_result.py

```diff
@@ -60,6 +60,11 @@ def n_oracle_calls(self):
         """Return the number of times the estimator was called."""
         return self._n_oracle_calls
 
+    @property
+    def oracle_calls_execution_time(self):
+        """Return the number of times the estimator was called."""
```

Comment:
I'm confused as to what this is. The name implies it's the time spent in the oracle (aka estimator.fit()) and will be a `TimeSpan`. However the the documentation is implying that it's an integer count of the number of occasions on which the oracle was called. I suspect the former is correct, but this should be cleared up.

Reply:
Oh copy-paste...